resources/definitions/zonestar_z8m3_printer.def.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "ZONESTAR Z8M3",
```

Comment:
I can't find any information about this printer. Can you provide a link to a spec sheet or purchasing page somewhere so that we can check this definition file?